copyist.go

```diff
@@ -215,6 +215,10 @@ func OpenSource(t testingT, source Source, recordingName string) io.Closer {
 			panic(r)
 		}
 
+		if currentSession.verificationErr != nil {
```

Comment:
nit: The TestSessionPanicsAreCaught test in copyist_test.go is testing the panic catching behavior of .Close, but it uses a code path that never panics. I think the test is still correct, but the name should be changed.

Reply:
Ah good call! Changed.